test/diagnostics/Localization/no_localization_files.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-typecheck-verify-swift -localization-path %S -locale en
+
```

Comment:
I think that this should be a warning just like in case of incorrect locale - there is a difference between missing directory/file and missing translations...

Reply:
- Do you think `DiagnosticFrontend.def` is suitable for such an error message?
- What about this error message `"cannot find YAML localization files for the specified locale code"`

Reply:
1. Sure!
2. We need multiple errors here:
   - when `path` doesn't exist something like - `specified localization directory '%0' does not exist, translation is disabled`
   - when directory exists but it doesn't have a file for a specified `locale` - `cannot find translations for '%0' at '%1': no such file`